docs/source/algorithms/IntegratePeaksShoeboxTOF-v1.rst

```diff
@@ -0,0 +1,86 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This is an algorithm to integrate single-crystal Bragg peaks in a :ref:`MatrixWorkspace <MatrixWorkspace>` by
+summing up counts in a shoebox of size ``NRows`` and ``NCols`` on the detector and ``NBins`` along the Time-of-flight
+(TOF) direction which can be specified in one of two ways:
+
+1. Provide ``NBins`` - number of TOF bins in the kernel
+
+2. Setting ``GetNBinsFromBackToBackParams=True`` and providing ``NFWHM`` - in which case ``NBins``  will be NFWHM x FWHM
+   of a :ref:`BackToBackExponential <func-BackToBackExponential>` peak at the center of each detector panel/bank at the
+   middle of the spectrum.
+
+Note to use method 2, back-to-back exponential coefficients must be defined in the Parameters.xml file for the
+instrument.
+
+The integration requires a background shell with negative weights, there are approximately the same number of bins in
+the background shell as in the peak region.
+
+The algorithm proceeds as follows:
+
+1. Take a window of the data, ``NShoeboxInWindow`` the size of the shoebox (``NRows`` x ``NCols`` x ``NBins``)
```

Comment:
Is it possible to add some lines explaining how the default shoebox is selected?